### PR TITLE
Cnft1 2292 sign up sign in screens

### PR DESCRIPTION
**Pull request by hclarkEnq**
## Description

Creating Sign in login component and leaving a placeholder when redirecting to the sign up screen

## Tickets

* [CNFT1-2292](https://cdc-nbs.atlassian.net/browse/CNFT1-2292)

## Checklist before requesting a review
- [x] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

[CNFT1-2292]: https://cdc-nbs.atlassian.net/browse/CNFT1-2292?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/modernization-ui/src/welcomeLayout/WelcomeLayout.tsx [hunk 1]
```diff
@@ -3,8 +3,16 @@ import { WelcomeHeader } from './WelcomeHeader/WelcomeHeader';
 import style from './welcomeLayout.module.scss';
 import { AlertBanner } from 'apps/page-builder/components/AlertBanner/AlertBanner';
 import { TabNavigationEntry, TabNavigation } from 'components/TabNavigation/TabNavigation';
+import { SignIn } from 'apps/landing/SignIn/SignIn';
+import { useState } from 'react';
+
+export type WelcomProps = {
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/welcomeLayout/WelcomeLayout.tsx:9]: question: Should this be in `SignIn.tsx`?

File: apps/modernization-ui/src/apps/landing/SignIn/SignIn.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            <h2 className={classNames(styles.heading)}>Login</h2>
+            <div className={classNames(styles['alert-banner'])}>
+                <AlertBanner type="info">
+                    <p>
+                        Please be sure to avoid entering any real PHI/PIl data on the demo site. All information entered
+                        will be viewable by other users.
+                    </p>
+                </AlertBanner>
+            </div>
+            <NavLinkButton to="/login" className="margin-top-2">
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/apps/landing/SignIn/SignIn.tsx:20]: issue:  The route `/login` is for development use only.  This should be pointing to the NBS 6 Login page at `/nbs/login`

**Review by adamloup-enquizit — APPROVED** [review]
🎈<sub>🎈</sub> <sup>🎈</sup><sup><sup>🎈</sup></sup>

**Review by stevegsa — APPROVED** [review]
🚀🚀🚀

**Pull request merged**